src/interface/api_swagger.json

```diff
@@ -2438,21 +2345,6 @@
       "maxLength": 128,
       "description": "Extension value."
     },
-    "FirstName": {
```

Comment:
I'm looking only at the changes here, but the FirstName, LastName, Name, MiddleName, FspId are things we need in /participants and /parties calls, isn't it?

Reply:
I have referenced these correctly